### PR TITLE
[TextFields] Text area comments, updated super, naming, formatting.

### PR DESCRIPTION
**Pull request by willlarche**

### CODE REVIEW

File: components/TextFields/src/MDCTextInputControllerOutlinedTextArea.h [hunk 1]
```diff
@@ -32,8 +32,39 @@
...
+
+ Normal Color - Black, 54% opacity
+
+ Rounded Corners - All
+
+ Trailing Underline Label Text Color - Black, 54% opacity
+
+ Underline Color Normal - Black, 54% opacity
+
+ Underline Height Active - 2p
```
Comment (romoore) [inline components/TextFields/src/MDCTextInputControllerOutlinedTextArea.h:62]: Is "2p" the same as Material's "dp"?

File: components/TextFields/src/MDCTextInputControllerOutlinedTextArea.h [hunk 2]
```diff
@@ -32,8 +32,39 @@
...
+ Floating Placeholder Color Active - Blue A700
+ Floating Placeholder Color Normal - Black, 54% opacity
+
+ Inline Placeholder Color - Black, 54% opacity
+
+ Leading Underline Label Text Color - Black, 54% opacity
+
+ Normal Color - Black, 54% opacity
+
+ Rounded Corners - All
```
Comment (romoore) [inline components/TextFields/src/MDCTextInputControllerOutlinedTextArea.h:56]: Nit: Add radius value?

File: components/TextFields/src/MDCTextInputControllerOutlinedTextArea.m [hunk 3]
```diff
@@ -86,13 +86,35 @@ - (void)textInputDidLayoutSubviews {
...
+
+ MDCTextInputTextFieldOutlinedTextAreaHalfPadding +                   // Small padding
+ MDCTextInputTextFieldOutlinedTextAreaPaddingAdjustment               // Additional point (iOS specific)
+ placeholderEstimatedHeight                                           // Height of placeholder
+ MDCTextInputTextFieldOutlinedTextAreaHalfPadding +                   // Small padding
+ MDCTextInputTextFieldOutlinedTextAreaPaddingAdjustment               // Additional point (iOS specific)
+ MDCCeil(MAX(self.textInput.font.lineHeight,                          // Text field or placeholder
+             self.textInput.placeholderLabel.font.lineHeight))
+ MDCTextInputTextFieldOutlinedTextAreaHalfPadding                     // Padding to bottom of border rect
+ underlineLabelsOffset                                                // From super class.
```
Comment (romoore) [inline components/TextFields/src/MDCTextInputControllerOutlinedTextArea.m]: Nit: s/underlineLabelsOffset/underlineOffset/

based on the code

**Review by romoore — APPROVED** [review]
Consider nits, but not necessary to merge.

**Review by jgunaratne — APPROVED** [review]

File: components/TextFields/src/MDCTextInputControllerOutlinedTextArea.h (hunk 1 above)
Comment (willlarche) [reply components/TextFields/src/MDCTextInputControllerOutlinedTextArea.h:62]: `p` is points like iOS points.

File: components/TextFields/src/MDCTextInputControllerOutlinedTextArea.h (hunk 2 above)
Comment (willlarche) [reply components/TextFields/src/MDCTextInputControllerOutlinedTextArea.h:56]: Radius value is not currently parameterized. Created issue #2631

File: components/TextFields/src/MDCTextInputControllerOutlinedTextArea.m (hunk 3 above)
Comment (willlarche) [reply components/TextFields/src/MDCTextInputControllerOutlinedTextArea.m]: Updated with more clarity.

File: components/TextFields/src/MDCTextInputControllerOutlinedTextArea.h (hunk 1 above)
Comment (romoore) [reply components/TextFields/src/MDCTextInputControllerOutlinedTextArea.h:62]: Got it. I wasn't sure which unit it was describing. I've noticed Apple tends to spell out "points", while Material uses "dp" as their "device-independent pixel" unit on Android.

**Pull request merged**